__tests__/handleFormSubmit.test.js

```diff
@@ -15,8 +17,11 @@ describe("handleFormSubmit", () => {
     };
   });
 
+  beforeEach(() => {
+    executeScriptSpy = vi.spyOn(global.chrome.scripting, 'executeScript').mockResolvedValue();
```

Comment:
move this to the let that you created but change it to const and call vi.resetAllMocks in tests where you need it to avoid reassignment.